scripts/update-apk.sh

```diff
@@ -22,8 +22,10 @@ else
 	/bin/rm -f phimpme-dev-*.apk
 fi
 
-\cp -r ../app/build/outputs/apk/app-debug.apk app-debug.apk
-\cp -r ../app/build/outputs/apk/app-release-unsigned.apk app-release-unsigned.apk
+\cp -r ../app/build/outputs/apk/*/**.apk .
+\cp -r ../app/build/outputs/apk/debug/output.json debug-output.json
+\cp -r ../app/build/outputs/apk/release/output.json release-output.json
+\cp -r ../README.md .
```

Comment:
Unneeded

Reply:
Still not resolved

Reply:
@iamareebjamal what do I need to remove?

Reply:
README

Reply:
I purposely left the readme. I thought that the apk branch will look better with the readme. Right now the branch looks dull as there are just a few files. 
I was also planning to add a small section to the readme telling a user that the apk can directly be downloaded from the apk branch.

Reply:
Please tell me if it is alright or if I should remove the readme file.

Reply:
remove

Reply:
ok. On it.